### DISCUSSION TITLE
Creating UTM Virtualized Ubuntu VM

### DISCUSSION THREAD

**Discussion by Pikishti** [utmapp/UTM#5059] [UTM Help (macOS)] [answered]
### Discussed in https://github.com/utmapp/UTM/discussions/5044

<div type='discussions-op-text'>

<sup>Originally posted by **Pikishti** February 20, 2023</sup>
Hello everyone,

I am having issues with the booting of a VM that uses Ubuntu. I am using the UTM app in an M1 Macbook Pro. In the image you can see the settings for the VM machine and also the issue I am getting when trying to start it. 
<img width="1331" alt="UTM issue" src="https://user-images.githubusercontent.com/114188730/220202361-eaa000c9-b7cb-4377-81ab-8f6a2d5660f6.png">

Has anyone ever experienced this issue? How can I fix it?</div>

**Comment by gitblanc** [comment]
Check this blog: https://askubuntu.com/questions/954042/why-does-computer-show-device-mapping-table-on-efi-shell-after-ubuntu-installati

It is similar to what you are experiencing

**Comment by gitblanc** [comment]
You can also type: `exit` on the prompt you are given and then select boot manager and start from the virtual disk named SATA 0x0 (on the top right of the boot manager)
Then you should be able to start ubuntu normally.
If that doesn't work you try to reinstall ubuntu on another virtual machine

**Comment by osy** [comment, marked as answer]
You chose an AMD64 iso and an ARM64 architecture. You should have seen an error message pop up in the wizard warning you that this isn’t going to work.
